### PR TITLE
Add an example showing how to use ppx_jane

### PR DESCRIPTION
**Pull request by yminsky**
Since the ppx_deriving line isn't entirely obvious, and I haven't seen documented elsewhere.

### CODE REVIEW

File: doc/quick-start.rst [hunk 1]
```diff
@@ -59,6 +59,37 @@ And build it with:
...
+    (jbuild_version 1)
+    
+    (executables
+     ((names (hello_world))
+      (libraries (core))
+      (preprocess (pps (ppx_jane ppx_driver.runner)))
+     )
+     
+This ``hello_world.ml`` file:
+
```
Comment (rgrinberg) [inline doc/quick-start.rst]: Don't forget `.. code:: ocaml` here.

Comment (yminsky) [inline doc/quick-start.rst]: Fixed now. the Lwt example above is incorrectly in scheme, but I guess that's a separate PR.

**Comment by bmillwood** [comment]
Hmm. A little while ago I stopped including the ppx_driver.runner line, and my stuff still works. Are you sure it's still necessary?

**Comment by diml** [comment]
Since v0.9.1, it's no longer necessary. Technically, the only difference for users is that by adding `ppx_driver.runner` you get access to the `-reconcile` option as well as support for `[@@deriving_inline ...] ... [@@@end]`. Since these are quite advanced features, it's probably not worth advertising it as the default method. @yminsky I think it'd be better to just put `(pps (ppx_jane))`.

See [this section of the manual](http://jbuilder.readthedocs.io/en/latest/advanced-topics.html#using-a-custom-ppx-driver) for more details.

**Comment by yminsky** [comment]
Fixed.

**Comment by diml** [comment]
Thanks

**Pull request merged**